cloudinit/net/sysconfig.py

```diff
@@ -635,12 +636,9 @@ def _render_subnet_routes(cls, iface_cfg, route_cfg, subnets, flavor):
                             "Duplicate declaration of default "
                             "route found for interface '%s'" % (iface_cfg.name)
                         )
-                    # NOTE(harlowja): ipv6 and ipv4 default gateways
-                    gw_key = "GATEWAY0"
-                    nm_key = "NETMASK0"
-                    addr_key = "ADDRESS0"
-                    # The owning interface provides the default route.
-                    #
+                    # NOTE that instead of defining the route0 settings,
+                    # the owning interface provides the default route.
+
```

Comment:
Note that the 3 keys removed here were never used at all. I think linting previously never complained about it because they were also defined and then used in the else clause.